### PR TITLE
Implement source address mode 0x04.

### PR DESCRIPTION
**Pull request by Adminiuga**
deCONZ serial protocol versions 0x010b and newer support source address mode `0x04` which includes both short network address and IEEE address. This PR implements address mode 0x04 for `DeconzAddress` class.
image

### CODE REVIEW

File: zigpy_deconz/types.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import enum
 
+import zigpy.types as t
```
Comment (damarco) [inline zigpy_deconz/types.py]: I would prefer to not have any zigpy dependencies except for `zigbee.application`. What do you think?

Comment (Adminiuga) [reply zigpy_deconz/types.py]: Fine by me, but I'd like to have NWK class defined with Hex representation like it is in zigpy. Such a small thing, but makes tons of difference when sipping through debug logs.

Comment (Adminiuga) [reply zigpy_deconz/types.py]: Removed Zigpy types dependency.

**Comment by Adminiuga** [comment]
@damarco removed zigpy dependency. Ready for review.

**Pull request merged**